extensions/terminal-suggest/scripts/pullZshBuiltins.ts

```diff
@@ -7,10 +7,16 @@ import { exec } from 'child_process';
 import { promisify } from 'util';
 import * as fs from 'fs/promises';
 import * as path from 'path';
+import { platform } from 'os';
+
+if (platform() === 'win32') {
+	console.error('\x1b[31mThis command is not supported on Windows\x1b[0m');
+	process.exit(1);
+}
 
 const shortDescriptions: Map<string, string> = new Map([
 	['.', 'Source a file'],
-	// [':', ''],
+	[':', 'Null command'],
```

Comment:
I like bash's description of this better, ` No effect; the command does nothing.  A zero exit code is returned.`

Reply:
I want to avoid any punctuation, how about just "No effect"?